api4/bot.go

```diff
@@ -378,3 +379,46 @@ func deleteBotIconImage(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func convertBotToUser(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireBotUserId()
+	if c.Err != nil {
+		return
+	}
+
+	bot, err := c.App.GetBot(c.Params.BotUserId, true)
```

Comment:
Do we want to include deleted bots in the query?

Reply:
I checked this one again, it should be `false`. We don't include deleted bots in the legacy cli tool. 👍